style.css

```diff
@@ -0,0 +1,259 @@
+html * {
+  max-height: 1000000px;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap');
+
+.appbar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.appbar .status {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-right: 10px;
+  top: 10px;
+}
+
+.icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-betwee;
+  align-items: center;
+  flex: 1;
+}
+
+.icon {
+  padding: 5px;
+}
+
+.navbar-toggler {
+  border: 0 !important;
+}
+
+.navbar-toggler:focus {
+  border: 0;
+  box-shadow: none;
+}
+
+.navbar-light .navbar-toggler-icon {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'%3e%3cpath stroke='rgba%2832, 53, 237, 0.55%29' stroke-linecap='round' stroke-miterlimit='10' stroke-width='2' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e");
+  border: 0;
+}
+
+.navbar {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+.navbar .navbar-light .navbar-toggler-icon {
+  color: #6070ff;
+}
+
+.nav-container .navbar-brand {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.main {
+  display: flex;
+  height: 70%;
+  margin-top: 30%;
+}
+
+.mobile {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 5%;
+  margin-right: 10%;
+  margin-bottom: 10px;
+  margin-top: 15%;
+}
+
+#paragraph2 {
+  display: flex;
+  color: #6070ff;
+  font-weight: bold;
+}
+
+.mobile .heading {
+  display: flex;
+  justify-content: center;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  margin-left: 0%;
+}
```

Comment:
- [ ] Kindly make your text more similar to the Figma design and also make sure no letter is behind the `background-image`

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205414292-64bd3a36-fcd6-4e55-aa35-c15c23fed1e0.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205414269-0ebc6e85-2e5f-4d2b-81ca-14de10a90a5e.png)